### DISCUSSION TITLE
[Feature Request]: 站点管理-里面的站点支持修改站点名称

### DISCUSSION THREAD

**Discussion by c55kai** [jxxghp/MoviePilot#3371] [讨论]
### 当前程序版本

1.4.3

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

因为发现mp支持增加用户，给别人来添加订阅。当别人来添加订阅 选择站点时，不了解站点名称的意思，我就在想能不能改名，方便别人理解

**Comment by jxxghp** [comment]
@qwetrz007sh 既然不懂，又为何要在订阅上选站点？ 设定中定好就行了。
